src/server/dev.ts

```diff
@@ -25,6 +25,7 @@ const compiler = webpack(webpackConfig)
 const args = minimist(process.argv.slice(2))
 const withVirtualRobots = args['virtual-robots'] || false
 const nbsFile = args.play
+const broadcastAddress = args['broadcast-address'] || '10.1.255.255'
```

Comment:
Broadcast address still isn't quite the right name for this, as traditionally it's supposed to be a multicast address.

Reply:
so what would be the correct name? or, what would be a better name?

Reply:
Rather than the ridiculously long `--nuclearnet-address`. I say lets just go with `--address`. There probably isn't ever going to be another command-line param type of address.